helm/pay-wallet-values-dev.yaml

```diff
@@ -77,9 +77,14 @@ microservice-chart:
     REDIS_PORT: "6380"
     REDIS_SSL_ENABLED: "true"
     WEB_LOGGING_LEVEL: "off"
+    DEFAULT_LOGGING_LEVEL: "debug"
```

Comment:
Do we need `debug` by default?

Reply:
Changed in 997040d23000d61ba88ad0ed2db1b4baba7caad0